### ISSUE TITLE
Do not work on VirtualBox 5

### ISSUE THREAD

**Issue by robsonpeixoto** [azukiapp/azk#481]
```
$ VBoxManage --version
5.0.0r101573

$ azk vm  status -vvvv

/usr/local/Cellar/azk/0.14.4/node_modules/vboxmanage/lib/parse.js:97
      return JSON.parse("{" + line + "}");
                  ^
SyntaxError: Unexpected token @
    at Object.parse (native)
    at to_item (/usr/local/Cellar/azk/0.14.4/node_modules/vboxmanage/lib/parse.js:97:19)
    at Array.map (native)
    at Object.exports.machinereadable_list (/usr/local/Cellar/azk/0.14.4/node_modules/vboxmanage/lib/parse.js:99:30)
    at /usr/local/Cellar/azk/0.14.4/node_modules/vboxmanage/lib/machine.js:47:37
    at Object.callback (/usr/local/Cellar/azk/0.14.4/node_modules/vboxmanage/lib/command.js:72:16)
    at next (/usr/local/Cellar/azk/0.14.4/node_modules/vboxmanage/node_modules/async/lib/async.js:720:43)
    at /usr/local/Cellar/azk/0.14.4/node_modules/vboxmanage/node_modules/async/lib/async.js:24:16
    at ChildProcess.<anonymous> (/usr/local/Cellar/azk/0.14.4/node_modules/vboxmanage/lib/command.js:64:20)
    at ChildProcess.emit (events.js:98:17)
    at maybeClose (child_process.js:766:16)
    at Socket.<anonymous> (child_process.js:979:11)
    at Socket.emit (events.js:95:17)
    at Pipe.close (net.js:466:12)
```

**Comment by gullitmiranda** [comment]
@robsonpeixoto, unfortunately looks like VirtualBox API has changed in version 5.0. We need to investigate it further to add support to this version. Thank you for letting us know!

**Comment by robsonpeixoto** [comment]
Thanks @gullitmiranda

**Comment by dynaum** [comment]
:+1:

**Comment by fearenales** [comment]
@robsonpeixoto The fix for this issue is available in azk v0.14.6.
May you check if this worked for you?

Thank you for your support!
